torchvision/models/detection/transform.py

```diff
@@ -56,6 +62,17 @@ def normalize(self, image):
         return (image - mean[:, None, None]) / std[:, None, None]
 
     def resize(self, image, target):
+        if self.is_xla:
+            # Resize to a fixed size on TPU to avoid dynamicity.
+            new_h, new_w = 600, 800
+            image = torch.nn.functional.interpolate(
+                image[None], size=(new_h, new_w), mode='bilinear', align_corners=False)[0]
```

Comment:
Mind explaining what `image[None]` is doing here?

Reply:
TBH not sure, but it's the original behavior of the method if you scroll down further to line 89